tsl/src/compression/compression_scankey.c

```diff
@@ -482,9 +486,9 @@ build_update_delete_scankeys(Relation in_rel, List *heap_filters, int *num_scank
 
 static bool
 create_segment_filter_scankey(Relation in_rel, char *segment_filter_col_name,
-							  StrategyNumber strategy, Oid subtype, ScanKeyData *scankeys,
-							  int *num_scankeys, Bitmapset **null_columns, Datum value,
-							  bool is_null_check, bool is_array_op)
+							  StrategyNumber strategy, Oid subtype, Oid opcode,
+							  ScanKeyData *scankeys, int *num_scankeys, Bitmapset **null_columns,
+							  Datum value, bool is_null_check, bool is_array_op)
```

Comment:
This function has a little too many parameters, maybe we should make it take the `BatchFilter`?

Reply:
i had the same thought but not all callers have a batchfilter